react_frontend/src/AdobePremiereImport/helpers.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable global-require */
+/* eslint-disable no-undef */
+/* eslint-disable no-constant-condition */
+/* eslint-disable no-await-in-loop */
+
+export const importInPremiere = async ({ url, name }) => {
+  /**
+   * Initialize CSInterface
+   */
+  const { CSInterface } = require('@cep/csinterface')
+
+  const csInterface = new CSInterface()
+
+  /**
+   * Create temporary directory
+   */
+  const tempFolder = '/tmp/com.zorroa.scrounger.panel'
+
+  window.cep.fs.makedir(tempFolder)
+
+  const filePath = `${tempFolder}/${name}`
+
+  /**
+   * Open file
+   */
+  const fs = cep_node.require('fs')
```

Comment:
Are all `cep` things native Adobe functions that are accessible within their Premier browser?

Reply:
well, more or less

in practice it does what you think it does

in theory [Adobe Common Extensibility Platform (CEP)](https://www.adobe.io/apis/creativecloud/cep.html) is built around the [Chromium Embedded Framework](https://bitbucket.org/chromiumembedded/cef) ; who does what is not exactly clear to me!